assets/js/modules/adsense/datastore/base.test.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * `modules/adsense` base data store tests.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import {
+	createTestRegistry,
+	provideSiteInfo,
+} from '../../../../../tests/js/utils';
+import { MODULES_ADSENSE } from './constants';
+
+describe( 'modules/adsense base data store', () => {
+	let registry;
+	let store;
+
+	beforeEach( () => {
+		jest.resetModules();
+
+		registry = createTestRegistry();
+		provideSiteInfo( registry );
+	} );
+
+	it( 'does not define the admin page when unified dashboard is enabled', () => {
+		const { enabledFeatures } = require( '../../../features' );
+		enabledFeatures.add( 'unifiedDashboard' );
+
+		store = require( './base' ).default;
+		registry.registerStore( MODULES_ADSENSE, store );
+
+		expect( store.selectors.getAdminScreenURL() ).toBe(
```

Comment:
@techanvil, I believe this selector should be called using registry, shouldn't it?

```js
expect( registry.select( MODULES_ADSENSE ).geAdminScreenURL() ).toBe( ... );
```

Same applies for other tests and modules.

Reply:
I was wondering about this at the time, I actually started off with the registry version but noticed it could be simplified to a more direct test of the selector. What's the reason for calling it via the registry instead of directly? I'm happy to make the change but it would be useful to understand the reasoning behind it. :)

Reply:
In the meantime I have gone ahead and made the change(s) :+1:

Reply:
@techanvil we need to use the registry to verify that everything works together. Think of it as a kind of  "end-to-end" testing for selectors. Calling the selector via the registry also verifies that the selector and partial datastore are registered correctly in the registry, whilst calling the selector directly using the exported store object doesn't guaranty it.

Reply:
@eugene-manuilov that makes sense - thanks for the clarification :)